porch/Makefile

```diff
@@ -253,3 +254,17 @@ deploy-no-sa: deployment-config-no-sa
 
 .PHONY: push-and-deploy-no-sa
 push-and-deploy-no-sa: push-images deploy-no-sa
+
+.PHONY: run-in-kind
+run-in-kind:
```

Comment:
`kind` support is cool. Recently I ran into situation where I wanted to test using external function in function-runner in my dev setup and couldn't do so. This will help.

Reply:
Yeah, this is helpful for debugging the porch e2e tests locally.